src/assets/styles/layout/layout.scss

```diff
@@ -42,6 +29,10 @@ main {
     line-height: 1.4rem;
     font-size: 0.9375rem;
   }
+  
+  a {
+    text-decoration: underline;
+  }
```

Comment:
where is the text decoration set to 'none' for anchor tags?

Reply:
I'm not sure here! Not my code originally but I would guess this declaration conflicted with one in another stylesheet.

Reply:
Hmm. it just looks weird to set text decoration to underline when that's the default. 

I'd be deleting this and hunting for where it's set otherwise and setting a more specific (class-based) selector instead, but maybe there's an unknown side-effect I haven't accounted for.